packages/models/src/lib/table.ts

```diff
@@ -1,20 +1,21 @@
 import { z } from 'zod';
 import { primitiveValueSchema } from './implementation/schemas';
 
+export const tableAlignmentSchema = z.enum(['l', 'c', 'r'], {
+  description: 'Cell alignment (l:left, r:right, c:center)',
+});
+export type TableAlignment = z.infer<typeof tableAlignmentSchema>;
+
 export const tableHeadingSchema = z.object(
   {
-    key: z.string(),
+    key: z.string().optional(),
     label: z.string().optional(),
+    align: tableAlignmentSchema.optional(),
   },
   { description: 'Table heading' },
 );
```

Comment:
Making all properties optional enables empty objects (e.g. `[{}, {}, {}]` or `[{}, { key: 'foo' }, {}]`), which I wouldn't consider valid.

My suggestion in https://github.com/code-pushup/cli/issues/637#issuecomment-2071704785 would have `key` always required, but also supports string array for setting only alignments. Configuration examples would be:
- only keys: `[{ key: 'foo' }, { key: 'bar' }]`
- with labels: `[{ key: 'foo', label: 'Foo' }, { key: 'bar', label: 'Foo' }]`
- with labels and alignment: `[{ key: 'foo', label: 'Foo', align: 'left' }, { key: 'bar', label: 'Foo', align: 'center' }]`
- only alignment: `['left', 'center']`

Reply:
fixed

Reply:
done